todo/dto/deferred_details_dto.py

```
@@ -0,0 +1,9 @@
+from pydantic import BaseModel
+from datetime import datetime
+from todo.dto.user_dto import UserDTO
+
+
+class DeferredDetailsDTO(BaseModel):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)